src/hangman.py

```diff
@@ -27,9 +77,29 @@ def main():
         if correct == False:
```

Comment:
Should be shortened to:

```python
if not correct: ...
```
Conversely, use

```python
if correct: ...
```

rather than

```python
if correct ==  True: ...
```